qemu/tests/cfg/drive_mirror.cfg

```diff
@@ -50,6 +50,8 @@
             setup_local_nfs = yes
             boot_target_image = yes
         - on_iscsi:
+            only raw
+            no 2qcow2
             image_format_target = raw
```

Comment:
When use "no qcow2", it will use "2raw" only, so i think "image_format_target = raw" can be deleted here.